test.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
task: we used this for testing purposes but you can probably delete it for now. If we're going to store test data in the repo, it's better we do it as a test fixture and have a test use it.

Reply:
fixed